src/sprout/Helpers/AI/AiApiBase.php

```diff
@@ -0,0 +1,41 @@
+<?php
+namespace Sprout\Helpers\AI;
+
+/**
+ * Base class to ensure common functions for API tooling
+ */
+abstract class AiApiBase
```

Comment:
Would this be better as an interface? I don't see how implementations would be sharing any internal state.